src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-CSharp/Program.MinimalAPIs.OrgOrIndividualB2CAuth.cs

```diff
@@ -89,11 +89,10 @@
         var error = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
         throw new HttpRequestException($"Invalid status code in the HttpResponseMessage: {response.StatusCode}: {error}");
     }
-
     var forecast =  Enumerable.Range(1, 5).Select(index =>
         new WeatherForecast
         (
-            DateTime.Now.AddDays(index),
+            Date = DateOnly.FromDateTime(DateTime.Now.AddDays(index)),
```

Comment:
```suggestion
            DateOnly.FromDateTime(DateTime.Now.AddDays(index)),
```

I think we can probably remove the `Date` argument name here. Same across the rest of this file and the `src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-CSharp/Program.MinimalAPIs.WindowsOrNoAuth.cs`